packages/typespec-vscode/README.md

```diff
@@ -1,25 +1,58 @@
-# TypeSpec Language Support for VS Code
+# TypeSpec in Visual Studio Code
 
 [Repository](https://github.com/microsoft/typespec) | [Documentation](https://typespec.io/docs) | [Issues](https://github.com/microsoft/typespec/issues) | [Samples](https://github.com/microsoft/typespec/tree/main/packages/samples/specs)
 
-This provides provides [TypeSpec](https://github.com/microsoft/typespec) language support for VS Code.
+Using TypeSpec in Visual Studio Code with the extension enables efficient work. It utilizes the capabilities of VS Code to offer IntelliSense, syntax highlighting, code autocompletion, formatting, live diagnostics, rename refactoring, and go-to definition. The extension also provides a seamless experience for TypeSpec project setup and emitter configuration.
 
-![](https://raw.githubusercontent.com/microsoft/typespec/main/docs/images/vscode.gif)
+## Quick Start
 
-## Features
```

Comment:
Why do we remove `Features` chapter? I think it is good to give out highlight the features here.